src/no/runsafe/UserControl/database/PlayerKickLog.java

```diff
@@ -22,22 +25,39 @@ public ISchemaUpdate getSchemaUpdateQueries()
 		update.addQueries(
 			"CREATE TABLE player_kick_log (" +
 				"`name` varchar(255) NOT NULL," +
-				"`timestamp` datetime NULL," +
+				"`timestamp` datetime NOT NULL," +

```

Comment:
was this schema revision changed previously, or did something change in mysql since this was made?